website/source/docs/guides/minikube.md

```diff
@@ -0,0 +1,259 @@
+    ports:
+    - containerPort: 9002
+      name: http
+    env:
+    - name: COUNTING_SERVICE_URL
+      value: "http://localhost:9001"
+# ...
+```
+
+Within our `dashboard` application, we can access the `counting` service by communicating with `localhost:9001` as seen on the last line of this snippet. Here we are looking at an environment variable that is specific to the Go application running in a container in this Pod. Instead of providing an IP address or even a Consul service URL, we tell the application to talk to `localhost:9001` where our local end of the proxy is ready and listening. Because of the annotation to `counting:9001` earlier, we know that an instance of the `counting` service is on the other end.
+
+This is what is happening in the cluster and over the network when we view the `dashboard` service in the browser.
+
+-> TIP: The full source code for the Go-based web services and all code needed to build the Docker images are available in the [repo](https://github.com/hashicorp/demo-consul-101).
+
+## Task 3: Use Consul Connect
+
+### Step 1: Create an Intention that Denies All Service Communication by Default
+
+For a final task, let's take this a step further by restricting service communication with intentions. We don't want any service to be able to communicate with any other service; only the ones we specify.
+
+Begin by navigating to the _Intentions_ screen in the Consul web UI. Click the "Create" button and define an initial intention that blocks all communication between any services by default. Choose `*` as the source and `*` as the destination. Choose the _Deny_ radio button and add an optional description. Click "Save."
+
+![Connect Deny](/assets/images/guides/minikube-connect-deny.png "Connect Deny")
+
+Verify this by returning to the application dashboard where you will see that the "Counting Service is Unreachable."
+
+![Application is Unreachable](/assets/images/guides/minikube-connect-unreachable.png "Application is Unreachable")
+
+### Step 2: Allow the Application Dashboard to Connect to the Counting Service
+
+Finally, the easy part. Click the "Create" button again and create an intention that allows the `dashboard` source service to talk to the `counting` destination service. Ensure that the "Allow" radio button is selected. Optionally add a description. Click "Save."
+
+![Allow](/assets/images/guides/minikube-connect-allow.png "Allow")
+
+This action does not require a reboot. It takes effect so quickly that by the time you visit the application dashboard, you'll see that it's successfully communicating with the backend `counting` service again.
+
+And there we have Consul running on a Kubernetes cluster, as demonstrated by two services which communicate with each other via Consul Connect and an Envoy proxy.
+
+![Success](/assets/images/guides/minikube-connect-success.png "Success")
+
+We encourage you to clone this repository and try it out yourself with `minikube`.
```

Comment:
"...clone this repository..."

Is this perhaps wording from the demo consul repo? Given that you've just walked folks all the way through all of the steps and it included multiple repos, it would be cleaner to remove this bit.

Reply:
Fixed in a2d0b7c41